examples/erc20/aETH/debt-dai.xml

```diff
@@ -0,0 +1,241 @@
+            <!-- TODO: alternatively, consider using SELECT
+            <ethereum:call contract="LendingPool" function="getUserReserveData" as="e18"
+                           type="userReserveData" select="currentBorrowBalance">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+            -->
+            <ethereum:call function="getCurrentBorrowBalance" as="e18">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="principleBorrowBalance">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:label>
+            <ts:string xml:lang="en">Principle</ts:string>
+        </ts:label>
+        <ts:origins>
+            <!-- TODO: alternatively, consider using SELECT
+            <ethereum:call contract="LendingPool" function="getUserReserveData" as="e18"
+                           type="userReserveData" select="principleBorrowBalance">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+            -->
+            <ethereum:call function="getPrincipalBorrowBalance" as="e18">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="borrowRateMode">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:label>
+            <ts:string xml:lang="en">Borrowing Rate Mode</ts:string>
+        </ts:label>
+        <ts:origins>
+            <!-- TODO: consider if mapping is applicable here -->
+            <ethereum:call function="getBorrowRateMode" as="uint">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="borrowRate">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:label>
+            <ts:string xml:lang="en">Borrowing Rate Mode</ts:string>
+        </ts:label>
+        <ts:origins>
+            <!-- strange enough, they seem to be using e23 here and we don't have as="e23" -->
```

Comment:
@colourful-land e27 not e23: https://docs.aave.com/developers/developing-on-aave/important-considerations#ray-math

Reply:
@James-Sangalli comment seems to be wrong, but I think Weiwu's calculation is using e27:

```
const borrowRate = result[4].div(1e25, 10).toPrecision(4);
```

It's a percentage, so e27 * 100 -> e25.

Maybe just need to update the comment here.